client/src/js/services/grid/TransactionService.js

```diff
@@ -295,6 +295,35 @@ function TransactionService($timeout, util, uiGridConstants, bhConstants, Notify
     return true;
   };
 
+
+  /**
+   * @method transaction
+   *
+   * @description
+   * This method is used to check if a transaction is balanced by their record
+   */
+  function transaction(entity) {
```

Comment:
@lomamech, the name `validateTransaction()` was fine.  It's `dataEntity` that should be called `transaction`.  Because it _is_ a transaction, right?